base/changes.txt

```diff
@@ -6,6 +6,12 @@ to completeness or accuracy and it contains some references to files that are
 not part of the distribution.
 ================================================================================
 
+2025-02-01  Jonathan P. Spratte
+
+    * ltcmd.dtx (subsection{User functions}):
+    Generalise \cs{ProcessList} by using \cs{tl_map_tokens:nn} instead of
+    \cs{tl_map_function:nN}.
```

Comment:
```suggestion
    Generalise \ProcessList by using \tl_map_tokens:nn instead of
    \tl_map_function:nN.
```

Reply:
This change looks wrong to me:
surely the \cs are needed here!

Reply:
It's in changes.txt so \cs-like markup is not necessary. For example,
https://github.com/latex3/latex2e/blob/c93624f91ea5d9cad32b11ed674a0094d85fca27/base/changes.txt#L160-L168

Reply:
@car222222 this is changes.txt not typeset text

Reply:
Yes, I just noticed that myself!

Reply:
The first entry in that file prior to my additions used this markup and I copied it (in fact many entries by FMi in that file do that). Should we fix all of them as well?

Reply:
An interesting question! 
We could ask Frank if he meant to do this!

Reply:
not I didn't meant to do that. But I usually use a  \changes as a starting point for the text in changes.txt and obviously in a couple of cases over the years I didn't properly cleaned it up afterwards.

Could and should be changed, I guess.

Reply:
I committed @muzimuzhi's suggestion, the remainder of the file should be cleaned up in a separate PR/issue.

Reply:
agreed. I dealt with the changes.txt in a separate PR